src/Agent/NewRelic/Agent/Extensions/NewRelic.Agent.Extensions/Api/InternalApi.cs

```diff
@@ -263,16 +263,16 @@ public static void IgnoreApdex()
         ///   &lt;body&gt;
         ///   ...
         /// </code></example>
-        public static string GetBrowserTimingHeader()
+        public static string GetBrowserTimingHeader(string? nonce)
         {
-            return _agentApiImplementation?.GetBrowserTimingHeader() ?? string.Empty;
+            return _agentApiImplementation?.GetBrowserTimingHeader(nonce) ?? string.Empty;
         }
 
         /// <summary> (This method is obsolete) gets browser timing footer. </summary>
         ///
         /// <returns> An empty string. </returns>
         [Obsolete]
-        public static string GetBrowserTimingFooter()
+        public static string GetBrowserTimingFooter(string? _)
```

Comment:
If this is already marked as `Obsolete`, might not be worth bothering to update.

Reply:
I only updated it to stay in sync with the change above. This could be reverted if that were updated.